src/Qir/Runtime/build-qir-runtime.ps1

```diff
@@ -106,6 +110,8 @@ if (!(Test-Path $osQirDropFolder)) {
     New-Item -Path $osQirDropFolder -ItemType "directory"
 }
 $pattern | Foreach-Object { Copy-Item (Join-Path . bin $_) $osQirDropFolder -ErrorAction SilentlyContinue }
+Copy-Item (Join-Path $env:NATIVE_SIMULATOR "Microsoft.Quantum.Simulator.Runtime.dll") $osQirDropFolder -ErrorAction SilentlyContinue
+$env:PATH += ";" + $osQirDropFolder
```

Comment:
In Linux the `:` is used as the path separator. This line looks applicable for Win only.